boa/metrics/__init__.py

```diff
@@ -1,6 +1,16 @@
 """
-.. todo::
+################################################
+Metrics Overview & Advanced Usage
+################################################
 
-    Coming soon!
+
+Metrics are the objective functions we are optimizing over
+or functions that we are adding in as further constraints of our problem.
+
+Metrics can be specified in your configuration by detailing the name
+of one of the predefined :mod:`Metrics <boa.metrics.metrics>` in BOA
+by passing in the name of the metric into your configuration file
+
```

Comment:
We should add a note here that points to where users need to look if they are using custom metrics.

Reply:
And also add mention of this to the user guide. I don't see anywhere obvious that points out that you can pass your own metric through directly if you need more customization